package.json

```diff
@@ -21,6 +21,9 @@
     "test.watch": "stencil test --spec --e2e --watchAll",
     "generate": "stencil generate"
   },
+  "dependencies": {
+    "@oberd/oui": "file:../oui"
+  },
```

Comment:
@oberd/oui should not  be a dependency of @oberd/oui. An accident? Please remove.

Reply:
![Screenshot from 2019-12-03 12-12-07](https://user-images.githubusercontent.com/37966315/70077330-3380ac00-15c6-11ea-90b8-afc0830507af.png)

Reply:
Clarified, will remove